### PR TITLE
Fixed base path

### PR DESCRIPTION
**Pull request by deinok**
It's breaking my build.
Also notice that its pointing outside of the repo

### CODE REVIEW

File: admin-su/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <artifactId>base</artifactId>
     <groupId>org.sakaiproject</groupId>
     <version>19-SNAPSHOT</version>
-    <relativePath>../../pom.xml</relativePath>
+    <relativePath>../pom.xml</relativePath>
```
Comment (ottenhoff) [inline admin-su/pom.xml]: Should this be ../master/pom.xml instead?

Comment (deinok) [reply admin-su/pom.xml]: The <artifactId> of `./pom.xml` its called `base` while the <artifactId> of `./master/pom.xml` its called `master`.
So, my interpetation of what the original developer was trying to referenciate its to point `./pom.xml`.

Comment (ern) [reply admin-su/pom.xml]: The parent for this tool should be master:
```
    <parent>
        <groupId>org.sakaiproject</groupId>
        <artifactId>master</artifactId>
        <version>19-SNAPSHOT</version>
        <relativePath>../master/pom.xml</relativePath>
    </parent>
```

Comment (deinok) [reply admin-su/pom.xml]: Changed.
NVM, I think that `<version>` and `<relativePath>` can be deleted. (Not sure)

**Comment by deinok** [comment]
@bjones86 SAK-40128 its okey?

ref: https://github.com/sakaiproject/sakai/commit/2f9d9e0f57e66b7b20b99c640846bf266f7b6002
I think that is useless to have jira issues with this kind of hotfixes

**Comment by bjones86** [comment]
@deinok, I understand that you think it's not worthwhile, but this is the process the community has adopted and we must all abide by it.

Please create a new JIRA ticket, and link it to SAK-40128 as 'depends on', and indicate in the description that this fixes a problem introduced in SAK-40128.

Please then also include the SAK-????? ticket number as a prefix for your commits, and as a prefix for the PR title.

**Pull request closed without merging**